plugins/Hartree_Fock/EZFIO.cfg

```diff
@@ -33,3 +33,9 @@ doc: If true, skip the (inactive+core) --> (active) and the (active) --> (virtua
 interface: ezfio,provider,ocaml
 default: False
 
+[take_input_guess]
```

Comment:
If you don't want to use the current MOs, you should delete them by hand. It is too complicated.